subworkflows/local/utils_nfcore_phaseimpute_pipeline/main.nf

```diff
@@ -77,32 +80,92 @@ workflow PIPELINE_INITIALISATION {
     //
     validateInputParameters()
 
+    //
+    // Create fasta channel
+    //
+    genome = params.genome ? params.genome : file(params.fasta, checkIfExists:true).getBaseName()
+    if (params.genome) {
+        genome = params.genome
+        fasta  = getGenomeAttribute('fasta')
+        fai    = getGenomeAttribute('fai')
```

Comment:
oh I see, that's where you do it

Reply:
Is this working well with igenomes?

Reply:
Good question.
I should try.

Reply:
I should check that the possibility of no fai